server/src/main/java/keywhiz/utility/DSLContexts.java

```diff
@@ -40,12 +40,9 @@ public static DSLContext databaseAgnostic(DataSource dataSource) throws SQLExcep
     try (Connection conn = dataSource.getConnection()) {
       dialect = dialect(conn);
     }
-    if (dialect == SQLDialect.H2) {
-      return DSL.using(dataSource, SQLDialect.H2,
+    return DSL.using(dataSource, SQLDialect.H2,
```

Comment:
Oof, I goofed during the refactor. This was originally meant to be consume the `dialect` collected above- is `using` clever enough to just work it out for me?